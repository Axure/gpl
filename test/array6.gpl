void main()
{
	string[2][2] str;
	char[2][2] ch;
	int[2][2] num;
	str[0][1] = "here";
	ch[1][0] = 'h';
	num[1][1] = 7;
	print(str[0][1]);
	print(ch[1][0]);
	print(num[1][1]);
	str[0][1] = "there";
	ch[1][0] = 'm';
	num[1][1] = 8;
	print(str[0][1]);
	print(ch[1][0]);
	print(num[1][1]);
}