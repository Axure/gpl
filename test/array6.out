here
h
7
there
m
8
